docs/api/conventions.md

```diff
@@ -285,7 +285,7 @@ export default function PostRoute() {
 
 Nested routes can also contain dynamic segments by using the `$` character in the parent's directory name. For example, `app/routes/blog/$postId/edit.tsx` might represent the editor page for blog entries.
 
-See the [routing guide](../guides/routing.md) for more information.
+See the [routing guide][11] for more information.
```

Comment:
For internal links I think we can do similar treatment that I described for the anchor links above. Use the content of the link. You may be able to just use something like https://www.npmjs.com/package/@sindresorhus/slugify for this.